### PR TITLE
[12.0][MIG] purchase_delivery_split_date

### PR DESCRIPTION
**Pull request by bjeficent**
Migration of module purchase_delivery_split_date to version 12.0

### CODE REVIEW

File: purchase_delivery_split_date/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# Copyright 2017 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Purchase Delivery Split Date",
+    "version": "12.0.1.0.0",
+    "summary": "Allows Purchase Order you confirm to generate one Incoming "
+               "Shipment for each expected date indicated in the Purchase "
+               "Order Lines",
+    "author": u"Numérigraphe, Eficent, Odoo Community Association (OCA)",
```
Comment (nikul-serpentcs) [inline purchase_delivery_split_date/__manifest__.py]: `u` ??

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Minor Change

File: purchase_delivery_split_date/__manifest__.py (hunk 1 above)
Comment (bjeficent) [reply purchase_delivery_split_date/__manifest__.py]: When i tried to generate the readme with ```oca-gen-addon-readme```, i had problems with the accent (see the author Numérigraphe). So i had to put that 'u' to say that string is in unicode.

Comment (nikul-serpentcs) [reply purchase_delivery_split_date/__manifest__.py]: > When i tried to generate the readme with `oca-gen-addon-readme`, i had problems with the accent (see the author Numérigraphe). So i had to put that 'u' to say that string is in unicode.

@bjeficent IMO Change `Numérigraphe`  to `Numerigraphe` and try to generate the readme with `oca-gen-addon-readme`.

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional review

File: purchase_delivery_split_date/__manifest__.py (hunk 1 above)
Comment (bjeficent) [reply purchase_delivery_split_date/__manifest__.py]: Updates done.

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM :+1:

**Pull request merged**